.github/workflows/delete-published-images.yml

```diff
@@ -53,6 +53,7 @@ jobs:
         - vro-svc-bie-kafka
         - vro-xample-workflows
         - vro-cc-app
+        - vro-max-cfi-app
```

Comment:
nit: should this have the domain within the name? such as `vro-ee-max-cfi-app` for each domain recognition?

Reply:
Agree. Perhaps the `domain-ee` folder should be `domain-max-cfi`?